core/src/main/scala/ai/lum/odinson/lucene/index/OdinsonIndexWriter.scala

```diff
@@ -154,9 +154,9 @@ class OdinsonIndexWriter(
       OdinsonIndexWriter.PARENT_TYPE,
       Store.NO
     ))
-    metadata.add(new LuceneStoredField(DOC_ID_FIELD, d.id))
+    //metadata.add(new LuceneStoredField(DOC_ID_FIELD, d.id))
     // FIXME: remove
```

Comment:
I hope someone knows what this fixme is about.  Maybe it's obsolete.

Reply:
I believe you're looking at an old commit.  That was a leftover from debugging/experimenting.